content/en/network_monitoring/performance/setup.md

```diff
@@ -344,6 +344,44 @@ If you already have the [Agent running with a manifest][4]:
 [3]: https://app.datadoghq.com/organization-settings/api-keys
 [4]: /agent/kubernetes/
 {{% /tab %}}
+{{% tab "Operator" %}}
+[The Datadog Operator][1] is a way to deploy the Datadog Agent on Kubernetes and OpenShift. It reports deployment status, health, and errors in its Custom Resource status, and it limits the risk of misconfiguration thanks to higher-level configuration options.
+
+1. To enable Network Performance Monitoring in Operator, use the following configuration:
+
+```yaml
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: placeholder
+  namespace: placeholder
+spec:
+  # (...)
+  features:
+    networkMonitoring:
+      enabled: true
+  agent:
+    # (...)
+    systemProbe:
+      enabled: true
+```
+
+2. Add the following volumes to your manifest:
```

Comment:
I reviewed the operator and I don't think step 2 is necessary. It should be done for you. https://github.com/DataDog/datadog-operator/blob/1953406f4fddfef3bccc4e93a96193b71eacfb0c/controllers/datadogagent/utils.go#L1095-L1109

Reply:
Agree